### DISCUSSION TITLE
'Open New SQL File' is not supported when multiple objects are selected in Oracle Explorer > Database view.

### DISCUSSION THREAD

**Discussion by suvnoque** [eclipse-theia/theia#14463] [Improvements] [answered]
![image](https://github.com/user-attachments/assets/84f68382-ef9c-4f12-99b2-be23f33c87d9)
Oracle Developer Tools for VS Code extension
'Open New SQL File' is not supported when multiple objects are selected in Oracle Explorer > Database view. error occurs even there is only one object selected

**Comment by msujew** [comment, marked as answer]
Hey @suvnoque,

please create a bug report with the appropriate reproduction steps for this.
